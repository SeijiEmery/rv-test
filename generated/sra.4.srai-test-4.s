sll 	x1, x1, x2
srai 	x1, x1, 1
nop
nop
nop
nop
nop
ebreak
