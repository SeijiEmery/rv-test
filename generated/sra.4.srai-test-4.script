load /x 0 generated/sra.4.srai-test-4.hex
writereg 1 1
writereg 2 63
setpc 0
run 1004
readreg 1
readreg 2