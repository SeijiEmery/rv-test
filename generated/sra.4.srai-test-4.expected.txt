tests/sra.test.s:50
R1 = 13835058055282163712
R2 = 63
